setup-ci.sh

```diff
@@ -15,6 +15,12 @@ read s3regionendpoint
 echo 'Please enter the s3bucketname'
 read s3bucketname
 
+echo 'Please enter the users login name'
+read userusername
+
+echo 'Please enter the users passowrd'
+read userpassword
```

Comment:
Make this then match the name of the previous - i.e. openshift_password or whatever you decide to go with.